pkg/apis/events/v1alpha1/auditlogs_lifecycle.go

```diff
@@ -26,6 +26,11 @@ func (s *AuditLogsSourceStatus) GetCondition(t apis.ConditionType) *apis.Conditi
 	return auditLogsSourceCondSet.Manage(s).GetCondition(t)
 }
 
+// GetTopLevelCondition returns the top level condition.
+func (s *AuditLogsSourceStatus) GetTopLevelCondition() *apis.Condition {
```

Comment:
sorry, I wasn't familiar with this TopLevelCondition... What it would be in this case? The AuditsLogReady condition?

Reply:
It will get the internal "happy" condition which is also the "ready" condition of AuditLogsSource.